plugins/modules/synchronize.py

```diff
@@ -135,10 +135,10 @@
         that does not match the inventory user, you should set this parameter to C(no).
     type: bool
     default: yes
-  use_ssh_args:
+  ssh_args:
     description:
-      - Use the ssh_args specified in ansible.cfg. Setting this to `yes` will also make `synchronize` use `ansible_ssh_common_args`.
```

Comment:
This parameter is used in https://github.com/ansible-collections/ansible.posix/blob/a994a58b23f22269e91e3611772105296538fe09/plugins/action/synchronize.py#L208

Reply:
I could not see it in `module` and I could not use it in my playbooks.
`ssh_args` it's not in documentation, but I used it with successful.
The same error is [here](https://github.com/ansible/ansible/blob/stable-2.9/lib/ansible/modules/files/synchronize.py#L146).